tests/driver_hsc/app.config

```diff
@@ -0,0 +1,2 @@
+CONFIG_MODULE_HSC=y
+CONFIG_MODULE_XTIMER=y
```

Comment:
This file should be named `app.config.test`, if not it is assumed to be used always, and not only for the testing of the dependency modelling of Kconfig.

Reply:
Thanks :-)